client/layout/sidebar/item.jsx

```diff
@@ -54,15 +54,16 @@ export default function SidebarItem( props ) {
 
 	useEffect( expandSectionIfSelected, [ props.selected ] );
 
+	const linkProps = showAsExternal ? { target: '_blank', rel: 'noreferrer' } : {};
+
 	return (
 		<li className={ classes } data-tip-target={ props.tipTarget } data-post-type={ props.postType }>
 			<a
 				className="sidebar__menu-link"
 				onClick={ props.onNavigate }
 				href={ url }
-				target={ showAsExternal ? '_blank' : null }
-				rel={ isExternalLink ? 'noopener noreferrer' : null }
```

Comment:
Note: `noreferrer` includes `noopener` behavior as well according to the [HTML standard](https://html.spec.whatwg.org/multipage/links.html#link-type-noreferrer).